testsuite/openmodelica/linearization/test_dump_languages.mos

```diff
@@ -0,0 +1,178 @@
+// name:     test_dump_languages.mos
+// keywords: <...>
+// status:   correct
+// teardown_command: rm -rf *simple_test* output.log
+//
+// <insert description here>
+//
+
+loadFile("test_dump_languages.mo"); getErrorString();
+
+setCommandLineOptions("--linearizationDumpLanguage=modelica");
+linearize(simple_test, stopTime=0); getErrorString();
+readFile("linear_simple_test.mo"); getErrorString();
+
+setCommandLineOptions("--linearizationDumpLanguage=matlab");
+linearize(simple_test, stopTime=0); getErrorString();
+readFile("linear_simple_test.mo"); getErrorString();
+
+setCommandLineOptions("--linearizationDumpLanguage=julia");
+linearize(simple_test, stopTime=0); getErrorString();
+readFile("linear_simple_test.mo"); getErrorString();
+
+setCommandLineOptions("--linearizationDumpLanguage=python");
+linearize(simple_test, stopTime=0); getErrorString();
+readFile("linear_simple_test.mo"); getErrorString();
```

Comment:
You make different files right? Results below look correct though. Forgot to update this?
```suggestion
readFile("linear_simple_test.m"); getErrorString();

setCommandLineOptions("--linearizationDumpLanguage=julia");
linearize(simple_test, stopTime=0); getErrorString();
readFile("linear_simple_test.jl"); getErrorString();

setCommandLineOptions("--linearizationDumpLanguage=python");
linearize(simple_test, stopTime=0); getErrorString();
readFile("linear_simple_test.py"); getErrorString();
```

Reply:
yea i did forget to update